spec/mailers/publishers/vacancy_change_mailer_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+require "dfe/analytics/rspec/matchers"
+
+RSpec.describe Publishers::VacancyChangeMailer do
+  let(:email) { Faker::Internet.email(domain: TEST_EMAIL_DOMAIN) }
+  let(:vacancy) { create(:vacancy, contact_email: email) }
+  let(:publisher) { vacancy.publisher }
+
+  describe "#notify" do
+    let(:mail) { described_class.notify(vacancy:) }
+    let(:notify_template) { NOTIFY_PRODUCTION_TEMPLATE }
+    let(:some_content) { I18n.t("publishers.vacancy_change_mailer.notify.content").first }
```

Comment:
This appears to check that the email contains the first character of this string. Is that what you intended?